docs/standards.md

```diff
@@ -207,6 +208,36 @@ Things to take note of:
 
 `TODO`
 
+## Batch Scripts
+
+There is a set of batch scripts written in [batch_scripts.py](../scripts/batch_scripts.py). The motivation for these scripts is to provide crucial functionality for the hackathon process that is lacking in the admin UX. The goal is to extend the functionality to provide a full-fledged CLI client so that we can interface with the API without using the admin dashboard.
+
+### How to Run
+
+This script expects python3, and will not work in python2. Before you run the script, make sure that you have installed the following library:
+
+* [bson](https://pypi.org/project/bson/): `pip install bson`
+
+Then, run the following command:
+```python3 batch_scripts.py```
```

Comment:
It may be worthwhile to describe what the expected result is. It seems to be that the script queries user input until it receives a valid action. I feel like this would also give the 'actions' section below a bit more context.